proofs/cbmc/poly_invntt_tomont/Makefile

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = poly_invntt_tomont_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = poly_invntt_tomont
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/poly.c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/ntt.c
```

Comment:
ntt.c is not needed I think

Reply:
fixed in [d303353](https://github.com/pq-code-package/mldsa-native/pull/180/commits/d3033535a19f0b6b9a7b24fb8ab25af372b74952)